tools/bazel

```diff
@@ -12,4 +12,6 @@ else
   readonly bazel_version_bazelrc="$root_dir/bazel_6.bazelrc"
 fi
 
-exec "$BAZEL_REAL" "--bazelrc=$bazel_version_bazelrc" "$@"
```

Comment:
Can you open a PR with this change? Is disjoint from this PR.

Reply:
Yes, just trying to get this one green and then will split it up.